### PR TITLE
PP-382: Support for Postgres-9.3.12 on Windows platform

### PR DESCRIPTION
**Pull request by gajendrasharmagit**
#### Issue-ID

* **[PP-382](https://pbspro.atlassian.net/browse/PP-XXXX)**

#### Problem description
* PBS needs to support current Postgres version 9.3.12.

#### Cause / Analysis
* PBS needs to support current Postgres version 9.3.12. 

#### Solution description
* printjob_svr no longer needs to undef pid_t as there is no definition conflict of pid_t, with Postgres 9.3.12.
* .gitignore is updated with a few more auto generated files. Also, last submission accidentally removed a few auto-generated files on UNIX. Re-added.

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [x] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by bremanandjk** [comment]
Thanks for your explanation, I sign-off on these changes

File: .gitignore [hunk 1]
```diff
@@ -194,8 +194,13 @@ src/lib/Libpbs/ecl_resc_def_all.c
 src/lib/Libpbs/ecl_resv_attr_def.c
 src/lib/Libpbs/ecl_sched_attr_def.c
 src/lib/Libpbs/ecl_svr_attr_def.c
+src/include/stamp-h1
+src/lib/Libpython/pbs_ifl.i
+src/lib/Libpython/pbs_ifl.py
+src/lib/Libpython/pbs_ifl_wrap.c
 
 #Generated source files - Windows
+win_configure/include/pbs_version_win.h
```
Comment (subhasisb) [inline .gitignore]: You are adding pbs_version_win.h to .gitignore; however this file is actually still part of the repository. If this is indeed an autogenerated file, then it should be removed from the repo

**Comment by gajendrasharmagit** [comment]
Subhasis and Bream,
I have removed the legacy pbs_version_win.h. I have updated the template pbs_version.h.in and pre-build event script gen_pbs_version.bat. Please have a look.

**Comment by subhasisb** [comment]
Code looks good. I sign off.

Can you kindly attached a fresh windows buildlog which shows that it actually builds on windows from scratch (ie, without a backup/vestigial pbs_version.h lying around)?

**Comment by gajendrasharmagit** [comment]
Thanks. Updated the ticket with fresh build logs on Windows without vestigial pbs_version.h.

**Comment by subhasisb** [comment]
All in place. Merging code.

**Comment by gajendrasharmagit** [comment]
Thanks.

**Pull request merged**